spyder/plugins/projects.py

```diff
@@ -277,6 +282,31 @@ def close_project(self):
             self.clear()
             self.restart_consoles()
 
+    def delete_project(self):
+        """
+        Delete current project without deleting the files in the directory
+        """
+        if self.current_active_project:
+            path = self.current_active_project.root_path
+            buttons = QMessageBox.Yes|QMessageBox.No
+            answer = QMessageBox.warning(self, _("Delete"),
+                                 _("Do you really want "
+                                   "to delete <b>%s</b>?<br><br>"
+                                   "<b>Note:</b> This action will only delete "
+                                   "the project. The files are going to be "
+                                   "preserved on disk."
+                                   ) % osp.basename(path), buttons)

```

Comment:
In general is nicer to use [format](https://docs.python.org/3.5/library/string.html#format-examples)